src/renderer/components/SplittedChatListAndView.js

```diff
@@ -140,6 +155,8 @@ class SplittedChatListAndView extends React.Component {
       selectedChatId = this.state.selectedChatId = this.getInitiallySelectedChatId()
     }
 
+    this.markNoticedChatIfNeeded(selectedChatId)
```

Comment:
Is this needed or can we move this elsewhere? I'd rather not do ipc stuff inside a render.

Reply:
Not the end of the world if it has to stay :)

Reply:
it's needed, but only done if the freshMessagesCounter in the selectedChat is > 0. Maybe we can do this before render.

Reply:
Leave it, with a TODO comments. Lets figure out later.